app/tests/hold-tap/balanced/1/events.patterns

```diff
@@ -0,0 +1,4 @@
+s/.*hid_listener_keycode/kp/p
```

Comment:
Can we please give each of these directories better names than `1`, `2`, etc? I am assuming these are the different use cases from your diagramming work (Which is awesome!), but perhaps something like `1-tap-on-press-release-while-held` or somesuch.

Thoughts on adding a `README.md` file in the `app/tests/hold-tap/` directory that sets some context on these tests and the scenarios they cover? Can also link to the diagram files you added to guide people to understand these tests?

More generally: ROCK ON FOR ADDING ALL THESE TESTS.

Reply:
I committed both the PDF and open document format of the files to the tests/hold-tap folder. I'll add a readme and somewhat descriptive names.